examples/GPSClient/GPSClient.ino

```diff
@@ -0,0 +1,108 @@
+// Meshtastic Arduino GPS client
+// by Mike Schiraldi
+//
+// Get an Adafruit Feather M0 WiFi (or some other board, but then edit the pin config below)
+// and put your Meshtastic node in WiFi AP mode, then put your SSID and wifi password in arduino_secrets.h
+// and run this code with the serial monitor open. Your arduino should connect to the node, request information
+// on it and all other nodes it knows about, and print a report of what it finds.
```

Comment:
I'd make the explanation a little more pragmatic. Outlining what this does, vs how to run it.
Here's a great outline https://docs.arduino.cc/hacking/software/ArduinoStyleGuide (minus the timestamp and attribution)

Reply:
I agree about explaining more about what this does, and that style guide is a good link, but I'm also going to keep the how-to-run-it part. As the link says, "The most important users of Arduino are beginners and people who don't care about code, but about getting projects done."